boards/deprecated.cmake

```diff
@@ -165,7 +165,10 @@ set(efr32_radio_brd4170a_DEPRECATED
     slwrb4170a
 )
 set(efr32_radio_brd4180a_DEPRECATED
-    slwrb4180a
+    slwrb4180a@0
+)
+set(efr32_radio_brd4180b_DEPRECATED
```

Comment:
No board with this name previously existed, so there should not be any change to this file.